src/main/java/redis/clients/jedis/search/Schema.java

```diff
@@ -273,4 +278,33 @@ public String toString() {
           + ", separator='" + separator + "'}";
     }
   }
+
+  public static class VectorField extends Field {
+
+    public enum VectorAlgo {
+      FLAT,
```

Comment:
Add a test for FLAT.

Reply:
The same test is working with FLAT. Do you want me to do duplicate it? :sweat_smile:

Reply:
Yes